<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModuleTemperatures</class>
 <widget class="QWidget" name="ModuleTemperatures">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>186</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Module Temperatures</string>
  </property>
  <widget class="QPushButton" name="pushButtonCompute">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>150</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Compute</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCancel">
   <property name="geometry">
    <rect>
     <x>254</x>
     <y>150</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>341</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Truncate Image Borders (percent of image width)</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="truncateWidthSpinBox">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>9</y>
     <width>61</width>
     <height>26</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="value">
    <number>5</number>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>116</y>
     <width>411</width>
     <height>25</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QLabel" name="progressLabel">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>93</y>
     <width>87</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Computing...</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>41</y>
     <width>248</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Local Neighborhood Radius (meters)</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="neighborRadiusSpinBox">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>40</y>
     <width>61</width>
     <height>26</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="value">
    <number>7</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
